mobile/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
 
+  path_provider_ios:
```

Comment:
Should I commit the pubspec.lock and Podfile.lock too?

Reply:
Yeah I think that is fine.